src/components/Box/Box.tsx

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 import { makeStyles } from '../../styles';
 import { GetClasses } from '../../typeUtils';
 import { warning } from '../../utils';
-import colorOptions from './colorOptions';
+import colorOptions from '../../colors/colorOptions';
```

Comment:
I wasn't sure if this is the right location to store `colorOptions` in, but I wanted to keep it DRY and share between `<Box` and `<Stepper` 🤷🏻‍♂️

Reply:
Works for me! We can always move it when content or context changes in the future.